config/static/nodes/rpm-overrides/install-custom-rpm.yaml.envsubst

```diff
@@ -20,11 +20,15 @@ spec:
           [Unit]
           Description=Install custom runc
           # https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
-          After=network-online.target rpm-ostreed.service
+          After=network-online.target
+          # After=network-online.target
+          StartLimitIntervalSec=10
+          StartLimitBurst=30
```

Comment:
I don't think these are really required... or if we do decide we want it (e.g. prevent unlimited download of external file due to unlimited restart), then these values are not going to achieve the desired effect.

From `systemd.unit(5)`, these settings work together to inhibit further restarts if the service restarts *burst* times in *interval-sec* seconds.  But here we are downloading an external RPM, so it seems unlikely that we would ever hit the rate of 30 in 10 seconds (or 3 per second).

So I would suggest either remove the setting (it is for a hack after all, not for producion use), or else change these settings to "10 times in 5 minutes" or something like that.  Alternatively, under `[Service]` specify a high `RestartSec` to configure a longer delay between restart attempts (perhaps 1 minute).

Reply:
Thanks @frasertweedale for your comments. I misread the systemd documentation and I wanted to set the values for 30 max intents to restart and a delay of 10 seconds between the retries (my thoughts).

I like "10 times in 5 minutes". I will change it. I want to limit this to avoid the systemd.unit is restarting the systemd.unit if for some reason the command fails continuously. About RestartSec I will set this to 10 seconds, so that if the process fails, we don't have to await too much until the RPM package is tried to be installed again.

I have seen in PSI cluster that first time is always failing. I am trying to retrieve a traceback that I saw the last Friday, so it could point out directly to the root cause.

In short: If the systemd.unit is failing continuously, it will be restarted with a delay of 10 seconds from the last failure; moreover the systemd.unit will stop restarting if the threshold is reached for 10 restarts in 5 minutes, avoiding that this is restarting and downloading the RPM packages continuously.

Reply:
Thanks, the new values LGTM.   I think we can do a final review during collab review session today, then merge.